invenio_base/cli.py

```diff
@@ -45,49 +57,71 @@ def list_entrypoints(entry_point):
             # Store entry points.
             if group_name not in found_entry_points:
                 found_entry_points[group_name] = []
-            for ep in entry_points.values():
-                found_entry_points[group_name].append(str(ep))
+            # Convert the entry point to string before appending
+            for _, ep_value in entry_points.items():
+                found_entry_points[group_name].append(str(ep_value))
 
     for ep_group in sorted(found_entry_points.keys()):
         click.secho(f"{ep_group}", fg="green")
         for ep in sorted(found_entry_points[ep_group]):
             click.echo(f"  {ep}")
 
 
-@instance.command("migrate-secret-key")
-@click.option("--old-key", required=True)
-@with_appcontext
-def migrate_secret_key(old_key):
+@instance.command("migrate-secret-key")  # type: ignore[misc]
```

Comment:
what is the real warning|error which is ignored here?

Reply:
the type: ignore[misc] is needed because mypy can't understand how Click's decorators change the function at runtime. This is normal when using Click's decorators with type checking.